ci-operator/step-registry/openshift/upgrade/qe/test/disconnected/cert-rotation/openshift-upgrade-qe-test-disconnected-cert-rotation-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: openshift-upgrade-qe-test-disconnected-cert-rotation
+  steps:
+  - ref: cucushift-upgrade-helper-service-build-info
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-mirror-images-by-oc-adm
+  - ref: cucushift-upgrade-healthcheck
+  - ref: cucushift-upgrade-toimage
+  - ref: openshift-upgrade-qe-test-disconnected-cert-rotation
```

Comment:
Not exactly. This is the one for disconnected.

Reply:
Thanks for the comments. Actually, I didn't dig into it this morning. I created two steps because we have two upgrade chains, probably `openshift-upgrade-qe-test-cert-rotation` is sufficient. 

Please do let me know if that's the case, then I can correct this after rehearsal.

Reply:
The disconnected version has an extra step `cucushift-upgrade-mirror-images-by-oc-adm` that mirrors the upgrade target images which is needed by step `cucushift-upgrade-toimage` in disconnected clusters.

Reply:
By looking at the script content of the 2 steps, they seems the same. But I don't know the test and don't know if extra action is needed in cert rotation test of disconnected env. cc @xingxingxia might have comments.